docs/_documentation/aws-walkthrough.md

```diff
@@ -55,7 +55,9 @@ Next, you need to specify your AWS credentials to use - you can select one of th
     ```
     
 
- * Shared Credentials file - The `~/.aws/credentials` file stores your credentials based on a profile name 
+ * Shared Credentials file - The `~/.aws/credentials` file stores your credentials based on a profile name. You can
+     specify the profile using the `--aws-profile` flag or the `KUBICORN_AWS_PROFILE` environment variable, otherwise
+     the `default` profile will be used.
```

Comment:
There already exists a common `AWS_PROFILE` environment variable that other tools use. Would it perhaps be better to re-use that?

Reply:
@nrb Fixed :)